### PR TITLE
Finish book challenge but it's weird

### PR DESCRIPTION
**Pull request by sasianan19**

### CODE REVIEW

File: Sierra/challenge_14/book_challenge.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+In book_list_class.py:
+Instantiate another object of class Booklist called nyt_bestsellers
+Then, add 2 books of your choice from the New York Times best sellers lists to nyt_bestsellers using the .add() method
+You can find NYT books here: https://www.nytimes.com/books/best-sellers/
+
+In this script:
+Then, print out the books attribute of nyt_bestsellers
+'''
+# This doesn't work here but it works in the book_list_class.py script!
+#print(nyt_bestsellers.books)
```
Comment (waldenwang9966) [inline Sierra/challenge_14/book_challenge.py:102]: For this part, you are supposed to instantiate the booklist objects first, and then add books to it, and then print them out. So basically, there are three things you have to write in this section.

File: Sierra/challenge_14/book_list_class.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		# I couldn't remove the title if it was in self.books for the remove_title method
+		
+	def count_books(self):
+		"""Return the number of books currently in the booklist"""
+		print(len(self.books))
+
+	def remove_title(self, title):
+		"""Remove a book from the book list"""
+		if title in self.books:
+			self.books.remove(title)
```
Comment (waldenwang9966) [inline Sierra/challenge_14/book_list_class.py:24]: For the original design, you can loop through all the books, and then call  if book.title == title, then remove the book object. Something like this:
for book in self.books:
    if book.title == title: 
         self.book.remove(book)

File: Sierra/challenge_14/book_list_class.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+	def display_titles(self):
+		"""Print out all titles currently in the book list"""
+		pass
```
Comment (waldenwang9966) [inline Sierra/challenge_14/book_list_class.py:28]: Here, what you can do is to initialize an empty titles list, and then append book one by one, and then sort the titles using sort function.

**Review by waldenwang9966 — CHANGES_REQUESTED** [review]

File: Sierra/challenge_14/book_challenge.py (hunk 1 above)
Comment (sasianan19) [reply Sierra/challenge_14/book_challenge.py:102]: It says to do those steps in the book_list_class.py script so I thought I was supposed to do it in there /: that's where I did those steps

**Comment by sasianan19** [comment]
For the most recent commit, I fixed what I could but because of the way I added the title and author, my other methods in the book_list_class.py don't work the way they should. I understand how the logic is supposed to work, but when I tried the ways it's supposed to be written, they didn't work for mine. Idk if this makes sense, but I hope so lol.

**Pull request merged**